src/Illuminate/Routing/Router.php

```diff
@@ -381,7 +381,7 @@ public function group(array $attributes, $routes)
     protected function updateGroupStack(array $attributes)
     {
         if (! empty($this->groupStack)) {
-            $attributes = RouteGroup::merge($attributes, end($this->groupStack));
+            $attributes = $this->mergeWithLastGroup(...func_get_args());
```

Comment:
Maybe will be better to change like:
```suggestion
            $attributes = $this->mergeWithLastGroup($attributes);
```